### DISCUSSION TITLE
How can we add multiple css properties inside single selector?

### DISCUSSION THREAD

**Discussion by leamandeep** [tailwindlabs/tailwindcss#9421] [Help] [answered]
Hi Tailwind Team! I want to ask how can we multiple css properties inside single selector?

`<div className="hover:bg-red-400 hover:text-white ..."></div>`

But I want something like

`<div className="hover:(bg-red-400 text-white) ..."></div>`

Writing again and again same selector is painful.

**Comment by wongjn** [comment, marked as answer]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314
